Week2/task16.js

```diff
@@ -0,0 +1,19 @@
+console.log(`More JavaScript`);
+console.log(`
+Task 16 `); // to observe the difference between a = b and b = a.
+
+let o1 = { foo: 'bar' };
+let o2 = { foo: 'bar' };
+let o3 = o2;
+
+o2 = { animal: 'cat' };
+
+console.log(o3);
+console.log('no');
+
+o1 = { animal: 'dog' };
+console.log(o3);
+console.log('no');
+
+console.log('we can not write o2 = o3, since we already define o2 and we did not define o3 yet');

```

Comment:
You defined `o3` on line 7, so we can write `o2 = o3`, but if we change the value of `o2` will it change `o3` as well?

Reply:
On my understanding 
I should write line 7 instead of o3 = o2 twrite o2 = o3, that is why I gave an explanation like that.
No.